.github/workflows/import/converters.js

```diff
@@ -0,0 +1,168 @@
+        const { innerHTML } = col;
+        const div = dom.window.document.createElement('div');
+        div.innerHTML = innerHTML;
+        return div;
+      });
+      const div = dom.window.document.createElement('div');
+      div.append(...divs);
+      return div;
+    });
+
+    const div = dom.window.document.createElement('div');
+    div.className = toBlockCSSClassNames(nameRow.textContent).join(' ');
+    div.append(...divs);
+    table.parentElement.replaceChild(div, table);
+  });
+}
+
+function makePictures(dom) {
+  const imgs = dom.window.document.querySelectorAll('img');
+  imgs.forEach((img) => {
+    const clone = img.cloneNode(true);
+    clone.setAttribute('loading', 'lazy');
+    // clone.src = `${clone.src}?optimize=medium`;
+
+    let pic = dom.window.document.createElement('picture');
+
+    const srcMobile = dom.window.document.createElement('source');
+    srcMobile.srcset = clone.src;
+
+    const srcTablet = dom.window.document.createElement('source');
+    srcTablet.srcset = clone.src;
+    srcTablet.media = '(min-width: 600px)';
+
+    pic.append(srcMobile, srcTablet, clone);
+
+    const hrefAttr = img.getAttribute('href');
+    if (hrefAttr) {
+      const a = dom.window.document.createElement('a');
+      a.href = hrefAttr;
+      const titleAttr = img.getAttribute('title');
+      if (titleAttr) {
+        a.title = titleAttr;
+      }
+      a.append(pic);
+      pic = a;
+    }
+
+    // Determine what to replace
+    const imgParent = img.parentElement;
+    const imgGrandparent = imgParent.parentElement;
+    if (imgParent.nodeName === 'P' && imgGrandparent?.childElementCount === 1) {
+      imgGrandparent.replaceChild(pic, imgParent);
+    } else {
+      imgParent.replaceChild(pic, img);
+    }
+  });
+}
+
+function makeSections(dom) {
+  const children = dom.window.document.body.querySelectorAll(':scope > *');
+
+  const section = dom.window.document.createElement('div');
+  const sections = [...children].reduce(
+    (acc, child) => {
+      if (child.nodeName === 'HR') {
+        child.remove();
+        acc.push(dom.window.document.createElement('div'));
+      } else {
+        acc[acc.length - 1].append(child);
+      }
+      return acc;
+    },
+    [section]
+  );
+
+  dom.window.document.body.append(...sections);
+}
+
+// Generic docs have table blocks and HRs, but not ProseMirror decorations
+export function docDomToAemHtml(dom) {
+  convertBlocks(dom);
+  makePictures(dom);
+  makeSections(dom);
+
+  return dom.window.document.body.innerHTML;
+}
+
+function makeHast(mdast) {
+  const handlers = {
+    ...defaultHandlers,
+    gridTable: mdast2hastGridTablesHandler(),
+  };
+  const hast = mdast2hast(mdast, { handlers, allowDangerousHtml: true });
+  return raw(hast);
+}
+
+function removeImageSizeHash(dom) {
+  const imgs = dom.window.document.querySelectorAll('[src*="#width"]');
+  imgs.forEach((img) => {
+    img.setAttribute('src', img.src.split('#width')[0]);
```

Comment:
Hopefully these URLs are always formatted correctly. Otherwise, using a regex to `replace` the width hash and its value might be a safer option.

Reply:
Keeping it as is for parity with nexter https://github.com/da-sites/nexter/blob/1932d72cac6d74d457bad14869ffa0cd85fe335e/nx/utils/converters.js#L136